js/src/ui/Translate/translate.js

```diff
@@ -0,0 +1,30 @@
+// Copyright 2015, 2016 Ethcore (UK) Ltd.
+// This file is part of Parity.
+
+// Parity is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Parity is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Parity.  If not, see <http://www.gnu.org/licenses/>.
+
+import React, { Component, PropTypes } from 'react';
+import { Translate as I18NTranslate } from 'react-i18nify';
+
+export default class Translate extends Component {
```

Comment:
Not sure I'm in favour of wrapping foreign components just for the sake of wrapping them.

There's a few points why this might not be worth the additional bloat:

- The whole JavaScript ecosystem basically uses SemVer. It is very unlikely that the interface of `react-i18nify` is suddenly going to change. Still, in this case we can specify an exact version in `package.json` or use a shrinkwrap file.
- `Translate` is most likely *always* going to be a mirror of the `I18NTranslate` component. If the latter gets more functionality that is useful to us, `Translation` will have to be changed additionally. If it gets more functionality that *is not* useful to us, most likely `Transition` won't have to be changed anyways, so it didn't save work.
- I know it sounds very generic, but more (complex) code usually is more sources of bugs, more files to have an overview over, bigger diffs, etc.